pyspecdata/core.py

```diff
@@ -7437,37 +7433,50 @@ def latex(self):
                 location = retval.find(symbol)
         # }}}
         logger.debug(strm(r"trying to generate",self.function_string,
-            '\n',retval,'\n',[allsymb[x] for x in argsort(locations)],
+            '\n',retval,'\n',[allsymb[x] for x in np.argsort(locations)],
             '\n',printfargs))
         return retval
     def settoguess(self):
         'a debugging function, to easily plot the initial guess'
-        self.fit_coeff = real(self.guess())
+        self.fit_coeff = np.real(self.guess())
         return self
     def _taxis(self,taxis):
         r'You can enter None, to get the fit along the same range as the data, an integer to give the number of points, or a range of data, which will return with 300 points'
         if taxis is None:
             taxis = self.getaxis(self.fit_axis).copy()
         elif isinstance(taxis, int):
-            taxis = linspace(self.getaxis(self.fit_axis).min(),
+            taxis = np.linspace(self.getaxis(self.fit_axis).min(),
                     self.getaxis(self.fit_axis).max(),
                     taxis)
-        elif not isscalar(taxis) and len(taxis) == 2:
-            taxis = linspace(taxis[0],taxis[1],300)
+        elif not np.isscalar(taxis) and len(taxis) == 2:
+            taxis = np.linspace(taxis[0],taxis[1],300)
         return taxis
     def eval(self,taxis,set_what = None,set_to = None):
-        r'''after we have fit, evaluate the fit function along the axis taxis
+        r'''after we have fit, eval calculates the fit function along the axis taxis.
         set_what and set_to allow you to forcibly set_what a specific symbol to a
         specific value --> however, this does not affect the class, but only
-        the return value'''
+        the return value. The specific value defines the number of evenly spaced points
+        along the taxis.
+        
+        Parameters
+        ----------
+        set_what: specific symbol you wish to assign or have included in the evaluation
+        set_to: the specific value (int) you are assigning the symbol you included
+        int: number of evenly spaced points along the taxis along the fit
+
+        Returns
+        -------
+        newdata:nddata of the calculated fit function along the axis taxis with a set 
+        amount of points along the fit and specifically set symbols with values.
+        '''
         if isinstance(set_what, dict):
```

Comment:
@guinneak198 : this is not a properly formatted numpy-style docstring, and this is important to know -- so see [this](https://www.sphinx-doc.org/en/master/usage/extensions/example_numpy.html#example-numpy) for a guide on how to write one.

- You don't have a short one-line description at the start
- The Parameters and Returns are missing types and the description is improperly formatted.